README.md

```diff
@@ -0,0 +1,2 @@
+# 499TeamExercise
+StringSorterAsc: Sorting the given arraylist of msg by its content ascending.
```

Comment:
This will create a merge error because @caincdiy and I modified readme directly from master. I think your way is better, modifying with the feature on the feature branch, but because of our mistake you will need to pull master to your feature branch to resolve the merge conflicts before you can commit